Sources/CasePaths/CasePath.swift

```diff
@@ -78,7 +78,7 @@ public struct CasePath<Root, Value> {
   }
 }
 
-#if swift(>=5.5)
+#if canImport(_Concurrency)
```

Comment:
We don't need `#if canImport(_Concurrency) && compiler(>=5.5.2)` like one previously did in TCA, right?

Reply:
oh maybe that's the magical incantation we need to get this finally passing on CI 😤